howso/ablation.amlg

```diff
@@ -76,10 +76,27 @@
 			))
 		)
 
+		(assign (assoc
+			influential_cases (get (call !SingleReact react_kwargs) "influential_cases")
+		))
+
+        (if
```

Comment:
stylistic change: put the if and the condition on the same line and combine the comments into one on the line above the if statement